Makefile

```diff
@@ -234,6 +234,6 @@ release-major: check-release bumpversion-major release
 # Dependency targets
 
 .PHONY: check-deps
-checkdeps:
-	$(eval allow_list='numpy|pandas|scikit-learn|tqdm|torch|rdt')
+check-deps:
+	$(eval allow_list='numpy=|pandas=|scikit-learn=|tqdm=|torch=|rdt=')
```

Comment:
I added the equals because without it, other libraries with the same prefix were picked up (ie. pandas-vet)